TYPO3.Flow/Classes/TYPO3/Flow/Error/Debugger.php

```diff
@@ -175,7 +175,7 @@ protected static function renderObjectDump($object, $level, $renderProperties =
         $scope = '';
         $additionalAttributes = '';
 
-        if ($object instanceof \Doctrine\Common\Collections\Collection) {
+        if ($object instanceof \Doctrine\Common\Collections\Collection || $object instanceof \ArrayObject) {
```

Comment:
Hi Bastian,

this part has been merged already. That was a suggestion I made a few days/weeks ago. I think I messed it up by pulling the latest changes from neos and pushed it to my repo. Next time I will take care to provide a clean pull request.

Sorry!